ecosystem/manager.py

```diff
@@ -52,6 +52,63 @@ def recompile(self):
         """Recompile `members.json` from human-readable files."""
         self.dao.compile_json()
 
+    def build_website(self):
+        """Generates the ecosystem web page reading `members.json`."""
+        environment = Environment(loader=FileSystemLoader("ecosystem/html/"))
+        website_template = environment.get_template("webpage.html")
+        card_template = environment.get_template("card.html")
+        tag_template = environment.get_template("tag.html")
+        link_template = environment.get_template("link.html")
```

Comment:
Style suggestion: Maybe reduce number of variables by putting these in a dictionary or dataclass. Then you can do something like `templates.tags.render(...)`.

Reply:
Good idea! The function has a lot of local variables, so it's convenient to reduce the number of variables, leaving room for future changes. I'll make that change tomorrow. Thanks for the suggestion!